hybrid/hybrid_simulation.py

```diff
@@ -148,7 +149,12 @@ def __init__(self,
             power_sources[k.lower()] = power_sources.pop(k)
 
         if 'pv' in power_sources.keys():
-            self.pv = PVPlant(self.site, power_sources['pv'])
+            if 'pv_plant' in power_sources['pv']:
+                self.pv = power_sources['pv']['pv_plant']                       # User instantiated plant
+            elif 'tech_config' in power_sources['pv']:
```

Comment:
I wonder if there's a more explicit way to let users choose between creating a DetailedPVPlant vs a PVPlant?

Reply:
I was wondering this too... let's discuss.